web/src/engine/websites/ReadComicOnline.ts

```diff
@@ -13,19 +14,36 @@ function ChapterExtractor(anchor: HTMLAnchorElement) {
     };
 }
 
-function MangInfoExtractor(element: HTMLDivElement) { //needed because titles are truncated in link
-    const anchor = element.querySelector<HTMLAnchorElement>('a');
-    const dom = new DOMParser().parseFromString(element.getAttribute('title'), 'text/html');
-    return {
-        id: anchor.pathname,
-        title: dom.querySelector('p.title').textContent.trim()
-    };
-}
+const pageScript = `
+    new Promise( (resolve, reject) => {
+        const arrayName = document.documentElement.innerHTML.match(/(\\w+)\\.push\\(pth/)[1];
+        let tries = 0;
+        const interval = setInterval(function () {
+            try {
+                const imagearray = window[arrayName];
+                if (!Array.isArray(imagearray)) return;
+                const lastElement = imagearray[imagearray.length - 1];
```

Comment:
Maybe perform check on all elements not just the last one, something like
`imagearray.filter(link => !/blank.gif/i.test(link)).map(link => new URL(link))`

Reply:
They are sequentially done, if last id ok i assume previouses are ok but you are right.